src/libraries/System.IO.FileSystem/tests/File/Create.cs

```diff
@@ -219,9 +220,13 @@ public void LongFileName()
 
         [Fact]
         [PlatformSpecific(CaseSensitivePlatforms)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51371", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
         public void CaseSensitive()
         {
+            if (RuntimeInformation.RuntimeIdentifier.StartsWith("iossimulator"))
+            {
+                return;
```

Comment:
To skip a test we normally don't just return but throw a SkipTestException, that way it is correctly counted as skipped rather than passing and we can add an explanation.

```suggestion
                throw new SkipTestException("iOS/tvOS simulators run on a macOS host and are subject to the same case sensitivity behavior.");
```